index.js

```diff
@@ -13,9 +13,10 @@ function simpleGet (opts, cb) {
 
   if (opts.url) parseOptsUrl(opts)
   if (opts.headers == null) opts.headers = {}
+  if (opts.json) opts.headers['content-type'] = 'application/json'
   if (opts.maxRedirects == null) opts.maxRedirects = 10
 
-  var body = opts.body
+  var body = opts.json && opts.body ? JSON.stringify(opts.body) : opts.body
```

Comment:
You can remove `&& opts.body` since `JSON.stringify(undefined)` just returns `undefined`.